solutions/src/main/java/ru/sberbank/school/task02/FxConversionServiceImpl.java

```diff
@@ -0,0 +1,60 @@
+package ru.sberbank.school.task02;
+
+import ru.sberbank.school.task02.util.ClientOperation;
+import ru.sberbank.school.task02.util.Quote;
+import ru.sberbank.school.task02.util.Symbol;
+import ru.sberbank.school.task02.exception.FxConversionException;
+
+import java.math.BigDecimal;
+import java.util.ArrayList;
+import java.util.List;
+
+
+
+public class FxConversionServiceImpl implements FxConversionService{
+
+    ExternalQuotesService externalQuotesService;
+
+    FxConversionServiceImpl(ExternalQuotesService externalQuotesService){
+        this.externalQuotesService = externalQuotesService;
+    }
+
+    @Override
+    public BigDecimal convert(ClientOperation operation, Symbol symbol, BigDecimal amount) throws FxConversionException {
+
+        List<Quote> quotes = new ArrayList<>();
+        quotes = externalQuotesService.getQuotes(symbol);
+
+        if(quotes.size() == 0)
```

Comment:
&#39;if&#39; construct must use &#39;{}&#39;s.

*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a381355343>*